src/ostorlab/runtimes/lite_local/agent_runtime.py

```diff
@@ -193,6 +195,7 @@ def create_definition_config(self) -> docker.types.ConfigReference:
         """
         agent_definition = self._docker_client.images.get(self.agent.container_image).labels.get('agent_definition')
         config_name = f'config_definition_{self.image_name}__{self.runtime_name}'
+        config_name = hashlib.md5(config_name.encode()).hexdigest()
```

Comment:
Please inline the string in here, no need to put in a variable that gets reassigned, it looks incorrect.